uint8 POSE_CENTER=0     # coordinates starts in the center of the object
uint8 POSE_BASE=1       # coordinates starts in the base of the object
